src/test/words/case.test.js

```diff
@@ -1,33 +1,33 @@
-import assert from 'assert';
-import {fixCase} from "../../lib/words/case";
-import Locale from "../../locale/locale";
+import assert from 'assert'
+import { fixCase } from '../../lib/words/case'
+import Locale from '../../locale/locale'
 
 describe('Corrects accidental uPPERCASE\n', () => {
 	let testCase = {
-		"cAPSLOCK": "capslock",
-		"(cAPSLOCK)": "(capslock)",
-		"Central Europe and Cyrillic tests: aĎIÉUБUГ": "Central Europe and Cyrillic tests: aďiéuбuг",
-		"Hey, JEnnifer!": "Hey, Jennifer!",
+		cAPSLOCK: 'capslock',
```

Comment:
How come single word keys do not need to be in quotes?

Reply:
Good point, stripping/adding key quotes should be disabled by adding the `"quoteProps": "preserve"` rule